### ISSUE TITLE
[FEATURE REQ] Notification improvements

### ISSUE THREAD

**Issue by Silent-Remux** [autobrr/autobrr#1077]
Would like to see the following if at all possible, thanks for considering, loving the software!

- Ability to set notifications as filter action, would ideally allow for ability to control notification level on a per filter basis.
- Have notifications for important actions such as disconnections / reconnections / updates be sent at a higher priority level.

**Comment by zze0s** [comment]
Hi! Happy you like it!

> Ability to set notifications as filter action, would ideally allow for ability to control notification level on a per filter basis.

We have this already which is probably the same thing? https://github.com/autobrr/autobrr/issues/602

> Have notifications for important actions such as disconnections / reconnections / updates be sent at a higher priority level.

Could you extend on this a bit? Is there any specific Notificator using priorities? We just send it all, some of them different events.

**Comment by Silent-Remux** [comment]
> Hi! Happy you like it!
> 
> > Ability to set notifications as filter action, would ideally allow for ability to control notification level on a per filter basis.
> 
> We have this already which is probably the same thing? #602
> 
> > Have notifications for important actions such as disconnections / reconnections / updates be sent at a higher priority level.
> 
> Could you extend on this a bit? Is there any specific Notificator using priorities? We just send it all, some of them different events.

Yep, re #602 I must have missed that, definitely the same thing.

Well to me it seems like autobrr itself should send these sort of events (IRC connections/update) at a higher priority level (which you can control if using pushover for example). So I feel like all of those alerts should use a priority of 1 (or have the ability to choose your priority). If #602 was then implemented, I'd then like to be able to control the priority level of those on a per filter basis.

**Comment by zze0s** [comment]
Okay so it relates to Pushover then since Discord for example does not have the concept of priority? :+1:

**Comment by Silent-Remux** [comment]
> Okay so it relates to Pushover then since Discord for example does not have the concept of priority? 👍

Yes as it seems like the most versatile option at the moment and I have no intention of using discord for this purpose.
